packages/tools/src/stateManagement/segmentation/helpers/getSegmentationActor.ts

```diff
@@ -76,11 +77,12 @@ export function getSurfaceActorEntry(
   segmentationId: string,
   segmentIndex?: number | string
 ) {
-  return getActorEntry(viewportId, segmentationId, (actor) =>
-    // @ts-expect-error
-    actor.representationUID?.startsWith(
```

Comment:
this gave me so much pain... 4 hours to figure out what was going on

this is not good because it will match Surface-1 with Surface 10, 11 , 12, 13, 14, 15 etc, or Surface-2 with 21, 22, 23.

such a horrible bug because it was saying there's an actor for a segment when there isn't one